examples/images/ex_get_cutouts.py

```diff
@@ -24,3 +24,7 @@
     for image in results:
         print("Downloading %s..." % name)
         image.cachedataset(filename="NVSSimages/%s.fits" % name)
+
+# This file can be deletd
```

Comment:
Please do delete the file (and the other similar ones, especially the notebooks) instead of commenting in it. The ultimate aim is to get rid of the `examples` directory altogether.